examples/MessagesController.php

```diff
@@ -14,6 +14,9 @@
 
 class MessagesController extends Controller
 {
+    protected $viewSpace = 'messenger';
```

Comment:
These `xyzSpace` changes don't belong to your original charge and make the examples far too complex.

Reply:
I put those so the templates and controllers can have a base and re-usable concept because if you are using different objects you're going to use different controllers for those, which means different routes and maybe different templates but the inner workings are going to be the same

Reply:
I could remove it but that means it needs 2 copies of all the templates because they have routes inside them as well as sub-templates.

Reply:
It's primary about reducing the amount of changes that aren't needed for this feature.